xmrstak/backend/amd/autoAdjust.hpp

```diff
@@ -79,6 +79,8 @@ class autoAdjust
 		configEditor configTpl{};
 		configTpl.set( std::string(tpl) );
 
+		constexpr size_t byte2mib = 1024u * 1024u;
```

Comment:
Did you mean 2MiB or 1MiB here? Either the name or the value doesn't match.

Reply:
good point, I will change it to `byteToMiB`

Reply:
Just personally, 

But for all const like this, i always use naming style:
`<First Unit Name> Per [Number] <Second Unit Name>`

Sample:
`SecondPerMinute = 60`
`SecondPerDay = 24 * 3600`
`SecondPer5Day = 5 * 24 * 3600`
`BytePerMByte = 1024 * 1024`
`BytePer2MByte = 2 * 1024 *1024`